compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/diagnotic/DefaultDiagnosticListener.java

```diff
@@ -44,6 +44,7 @@ public void received(Diagnostic diagnostic) {
                 console.println("warning: " + pos + " " + diag.msg);
                 break;
             case NOTE:
+                // console.println("note: " + pos + " " + diag.msg);
```

Comment:
Looks like committed by mistake?

Reply:
This is actually to help stdlib folks to identify which functions can be marked as isolated. As discussed during the meeting, we can alternatively make this a warning, the issue with that though is that the build fails unless we special case that warning too.